platform_disk_api/api.py

```diff
@@ -248,9 +272,14 @@ async def handle_list_disks(self, request: Request) -> Response:
         tree = await self._auth_client.get_permissions_tree(
             username, self._disk_cluster_uri
         )
+        org_name = request.query.get("org_name")
+        project_name = request.query.get("project_name")
+        in_project = False if project_name == username else None
```

Comment:
I think this `in_project` logic should be pushed into `Service`. we have the same condition while creating a pvc and I guess it is better to keep such things relatively close. from the interface perspective we call `get_all_disks` and pass `project_name`, but then also specify `in_project`. maybe we could pass `user_name` or something there.